applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_element.hpp

```diff
@@ -300,6 +300,20 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) UPwSmallStrainElement : public UPwBa
                                     RetentionLaw::Parameters& rRetentionParameters,
                                     unsigned int              GPoint);
 
+    ///
+    /// \brief This function calculates the constitutive matrices, stresses and strains depending on the
+    ///        constitutive parameters. Note that depending on the settings in the rConstitutiveParameters
+    ///        the function could calculate the stress, the constitutive matrix, the strains, or a combination.
```

Comment:
The strain is only given because we always use the line `ConstitutiveParameters.Set(ConstitutiveLaw::USE_ELEMENT_PROVIDED_STRAIN);` on the ConstitutiveLaw::Parameters we pass to this function. If we don't the strain could actually be calculated by the constitutive law. Therefore, rStrainVectors is a non-const&, because at this point the function is generic and at this point I'd like this to work with any ConstitutiveLaw::Parameters and any constitutive law